cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/write_cloudwatch_agent_json.py

```diff
@@ -132,6 +137,44 @@ def select_logs(configs, args):
     return selected_configs
 
 
+def get_node_roles(scheudler_plugin_node_roles):
+    node_type_roles_map = {"ALL": ["ComputeFleet", "HeadNode"], "HEAD": ["HeadNode"], "COMPUTE": ["ComputeFleet"]}
+    return node_type_roles_map.get(scheudler_plugin_node_roles)
+
+
+def load_config(cluster_config_path):
+    with open(cluster_config_path) as input_file:
+        return yaml.load(input_file, Loader=yaml.SafeLoader)
+
+
+def add_scheduler_plugin_log(config_data, cluster_config_path):
+    """Add custom log files to config data if log files specified in scheduler plugin."""
+    cluster_config = load_config(cluster_config_path)
```

Comment:
--cluster-config-path is not required and might not be passed

Reply:
Added an if condition to check if it is pass and manually tested with create a batch cluster